2022 - Legiones Astartes.cat

```diff
@@ -24456,13 +24456,552 @@ A Perfect Fury: If a model or models with the Skill Unmatched special rule selec
+            </rule>
+          </rules>
+          <selectionEntries>
+            <selectionEntry id="59ed-c7c7-af5f-0772" name="Atomantic Deflector" hidden="false" collective="false" import="true" type="upgrade">
+              <constraints>
+                <constraint field="selections" scope="parent" value="1.0" percentValue="false" shared="true" includeChildSelections="false" includeChildForces="false" id="e743-9a98-00c8-1180" type="max"/>
+                <constraint field="selections" scope="parent" value="1.0" percentValue="false" shared="true" includeChildSelections="false" includeChildForces="false" id="d508-3d54-86ee-720b" type="min"/>
+              </constraints>
+              <profiles>
+                <profile id="573e-36e1-d1ef-069f" name="Atomantic Deflector" publicationId="a716-c1c4-7b26-8424" page="140" hidden="false" typeId="2a1f-7837-f0ef-be44" typeName="Wargear Item">
+                  <characteristics>
+                    <characteristic name="Description" typeId="347e-ee4a-764f-6be3">A model with an atomantic deflector gains a 5+ Invulnerable Save and any model with an atomantic deflector and a Wounds Characteristic that suffers an unsaved Wound with the Instant Death special rule is not immediately removed as a casualty, but instead loses D3 Wounds instead of one for each unsaved Wound with the Instant Death special rule inflicted on it. In addition, when a model with an atomantic deflector loses its last Wound or Hull Point, but before it is removed as a casualty or replaced with a Wreck, all models both friendly and enemy within D6&quot; suffer an automatic Hit at Str 8, AP -.</characteristic>
+                  </characteristics>
+                </profile>
+              </profiles>
+              <costs>
+                <cost name="Pts" typeId="d2ee-04cb-5f8a-2642" value="0.0"/>
+              </costs>
+            </selectionEntry>
+          </selectionEntries>
+          <selectionEntryGroups>
+            <selectionEntryGroup id="5f5a-bc15-c9d5-e890" name="May take one of the following:" hidden="false" collective="false" import="true">
+              <constraints>
+                <constraint field="selections" scope="parent" value="0.0" percentValue="false" shared="true" includeChildSelections="false" includeChildForces="false" id="5584-5c85-0695-6f61" type="min"/>
+                <constraint field="selections" scope="parent" value="1.0" percentValue="false" shared="true" includeChildSelections="false" includeChildForces="false" id="79ac-e95d-f127-67e6" type="max"/>
+              </constraints>
+              <selectionEntries>
+                <selectionEntry id="77eb-0040-30ff-2f20" name="Phosphex Discharger" hidden="false" collective="false" import="true" type="upgrade">
```

Comment:
Link to the shared selection entry of this option. If it doesn't exist already, create it as a shared selection entry.